src/Packetery/Module/Checkout.php

```diff
@@ -375,6 +382,7 @@ public function updateOrderMeta( int $orderId ): void {
 		}
 
 		if ( $this->isPickupPointOrder() ) {
+			$wcOrder = wc_get_order( $orderId );
```

Comment:
$wcOrder může být false, to je třeba ošetřit.

Reply:
$orderId předává woocomerce poté co vytvoří objednávku. Listener se používá k aktualizaci metadat objednávky. False se v tomto případě nikdy vrátí. Jakou situaci máš na mysli?

Reply:
Rozumím, ale raději bych tam viděl if navíc. Vzhledem k tomu, že jde o kód volaný hookem, nejde to při kontrole kódu pohodlně ověřit.